en/docs/setup/migrating-preparing-for-migration.md

```diff
@@ -21,6 +21,113 @@
 
 With WSO2 Identity Server 5.11.0, groups and roles are separated. For more information, see [What Has Changed in 5.11.0](../../setup/migrating-what-has-changed#group-and-role-separation).
 
+## Groups and Roles Improvements Migration
+
+With WSO2 Identity Server 5.12.0, groups and roles improvements are introduced. For more information, see [What Has Changed in 5.11.0](../../setup/migrating-what-has-changed#roup and role separation Improvements).
+
+Please note that the abbreviations mentioned in that section is also used in here.
```

Comment:
IMO better to mention this inline.
https://github.com/wso2/docs-is/pull/2603/files#diff-887c248893fa847c87617c31411e22667ecce022d8bb7a38ffa260c5c14c8caeR82-R86

As a practice, we try to avoid as many page navigations as possible to provide a better experience to readers.

Reply:
Fixed with fa6da06b8ecc8fd6ac34122f47ef8e9050296214